backend/grpc/protobuf/data.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package datapb;
+
+message DataRequest {
+    string filename = 1;
+}
+
+message Segment {
```

Comment:
Please, comment on which fields are the key of the segment and how to interpret inventories.

Reply:
- Rename inventories to inventory_volumes
 - Rename Segment to SegmentData
 - Comment above the field instead of end of the line.
 - Avoid block comments. Use // instead.